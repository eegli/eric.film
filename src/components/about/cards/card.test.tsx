import { render } from '@/src/tests/testUtils';
import Card from './card.component';

describe('About', () => {
  it('renders card component', () => {
    const { asFragment } = render(<Card />);
    expect(asFragment()).toMatchSnapshot();
  });
});
