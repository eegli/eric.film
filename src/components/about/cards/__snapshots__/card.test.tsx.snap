// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders card component 1`] = `
.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c7 {
  margin: 0 0.5rem;
  font-size: 18px;
  color: #f7f7f7;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-bottom: 1.2rem;
}

.c0 {
  font-style: italic;
  -webkit-letter-spacing: 0.2rem;
  -moz-letter-spacing: 0.2rem;
  -ms-letter-spacing: 0.2rem;
  letter-spacing: 0.2rem;
  color: #f7f7f7;
  font-weight: normal;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  max-width: 350px;
  margin: 0.5rem;
}

.c4 {
  -webkit-letter-spacing: 0.5rem;
  -moz-letter-spacing: 0.5rem;
  -ms-letter-spacing: 0.5rem;
  letter-spacing: 0.5rem;
  font-weight: 400;
  text-align: center;
  margin: 1rem 0 1rem 0;
  font-size: 2rem;
  color: #f7f7f7;
}

.c5 {
  background-color: #222222;
  padding: 2rem;
}

.c5 span {
  color: #f7f7f7;
  font-size: 14px;
}

.c5 p {
  margin: 0.5rem 0 0.5rem 0;
}

.c5 li {
  margin-bottom: 0.5rem;
}

.c3 {
  max-width: 100%;
  max-height: 100%;
}

@media (max-width:700px) {
  .c5 {
    padding: 1rem;
  }
}

<ThemeProvider
  theme={
    Object {
      "breakpoints": Object {
        "large": "1200px",
        "medium": "960px",
        "small": "830px",
        "xsmall": "700px",
      },
      "colors": Object {
        "accents": Object {
          "brightBlue": "#365952",
          "eggshell": "#53272D",
          "melon": "#034363",
        },
        "bgPrimary": "#121212",
        "bgSecondary": "#222222",
        "bgTertiary": "#3b3b3b",
        "pink": "#feb8ce",
        "textPrimary": "#f7f7f7",
        "textSecondary": "#E0E0E0",
        "textTertiary": "#8F8F8F",
      },
      "fontsizes": Object {
        "baseFontSize": "16px",
        "baseIconFontSize": "18px",
        "blogFontSize": "18px",
        "h1FontSize": "22px",
        "h2FontSize": "20px",
        "secondaryFontSize": "14px",
      },
      "transpaparency": Object {
        "opa10": "1A",
        "opa20": "33",
        "opa30": "4D",
        "opa40": "66",
        "opa50": "80",
        "opa60": "99",
        "opa70": "BF",
        "opa80": "CC",
        "opa90": "E6",
        "opa95": "F2",
      },
    }
  }
>
   
  <CardComponent>
    <cardstyles__CardHeader>
      <h1
        className="c0"
      >
         meet the amazing team 
      </h1>
    </cardstyles__CardHeader>
    <cardstyles__CardWrapper>
      <div
        className="c1"
      >
        <cardstyles__Card>
          <div
            className="c2"
          >
            <cardstyles__CardPicture>
              <picture
                className="c3"
              >
                <source
                  srcSet="/static/img/about/about-2.webp"
                  type="image/webp"
                />
                <img
                  alt="logo"
                  src="/static/img/about/about-2.jpg"
                />
              </picture>
            </cardstyles__CardPicture>
            <cardstyles__CardTitle>
              <h1
                className="c4"
              >
                eric
              </h1>
            </cardstyles__CardTitle>
            <cardstyles__CardDescription>
              <div
                className="c5"
              >
                <span>
                  FOUNDER
                </span>
                <p>
                  eric has been with us from the start. he is the founding father of eric.film and oversees all business activities as well as HR-related topics. Main responsibilities:
                </p>
                <ul>
                  <li>
                    project management
                  </li>
                  <li>
                    photography
                  </li>
                </ul>
                <SocialBar>
                  <social-barstyles__Container>
                    <div
                      className="c6"
                    >
                      <social-barstyles__IconContainer
                        aria-label="Link to Github"
                        href="https://github.com/eegli"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Github"
                          className="c7"
                          href="https://github.com/eegli"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaGithub>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 496 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 496 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaGithub>
                        </a>
                      </social-barstyles__IconContainer>
                      <social-barstyles__IconContainer
                        aria-label="Link to Linkedin"
                        href="https://www.linkedin.com/in/ericegli/"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Linkedin"
                          className="c7"
                          href="https://www.linkedin.com/in/ericegli/"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaLinkedinIn>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 448 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 448 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaLinkedinIn>
                        </a>
                      </social-barstyles__IconContainer>
                      <social-barstyles__IconContainer
                        aria-label="Link to Instagram"
                        href="http://instagram.com/eric.film"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Instagram"
                          className="c7"
                          href="http://instagram.com/eric.film"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaInstagram>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 448 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 448 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaInstagram>
                        </a>
                      </social-barstyles__IconContainer>
                    </div>
                  </social-barstyles__Container>
                </SocialBar>
              </div>
            </cardstyles__CardDescription>
          </div>
        </cardstyles__Card>
        <cardstyles__Card>
          <div
            className="c2"
          >
            <cardstyles__CardPicture>
              <picture
                className="c3"
              >
                <source
                  srcSet="/static/img/about/about-1.webp"
                  type="image/webp"
                />
                <img
                  alt="logo"
                  src="/static/img/about/about-1.jpg"
                />
              </picture>
            </cardstyles__CardPicture>
            <cardstyles__CardTitle>
              <h1
                className="c4"
              >
                eric
              </h1>
            </cardstyles__CardTitle>
            <cardstyles__CardDescription>
              <div
                className="c5"
              >
                <span>
                  LEAD DEVELOPER
                </span>
                <p>
                  eric is our self-taught javascript engineer. he mostly talks about "react" but as long as our website looks good, we don't really care. Main responsibilities:
                </p>
                <ul>
                  <li>
                    eric.film development
                  </li>
                  <li>
                    blogging
                  </li>
                </ul>
                <SocialBar>
                  <social-barstyles__Container>
                    <div
                      className="c6"
                    >
                      <social-barstyles__IconContainer
                        aria-label="Link to Github"
                        href="https://github.com/eegli"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Github"
                          className="c7"
                          href="https://github.com/eegli"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaGithub>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 496 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 496 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaGithub>
                        </a>
                      </social-barstyles__IconContainer>
                      <social-barstyles__IconContainer
                        aria-label="Link to Linkedin"
                        href="https://www.linkedin.com/in/ericegli/"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Linkedin"
                          className="c7"
                          href="https://www.linkedin.com/in/ericegli/"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaLinkedinIn>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 448 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 448 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaLinkedinIn>
                        </a>
                      </social-barstyles__IconContainer>
                      <social-barstyles__IconContainer
                        aria-label="Link to Instagram"
                        href="http://instagram.com/eric.film"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Instagram"
                          className="c7"
                          href="http://instagram.com/eric.film"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaInstagram>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 448 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 448 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaInstagram>
                        </a>
                      </social-barstyles__IconContainer>
                    </div>
                  </social-barstyles__Container>
                </SocialBar>
              </div>
            </cardstyles__CardDescription>
          </div>
        </cardstyles__Card>
        <cardstyles__Card>
          <div
            className="c2"
          >
            <cardstyles__CardPicture>
              <picture
                className="c3"
              >
                <source
                  srcSet="/static/img/about/about-3.webp"
                  type="image/webp"
                />
                <img
                  alt="logo"
                  src="/static/img/about/about-3.jpg"
                />
              </picture>
            </cardstyles__CardPicture>
            <cardstyles__CardTitle>
              <h1
                className="c4"
              >
                eric
              </h1>
            </cardstyles__CardTitle>
            <cardstyles__CardDescription>
              <div
                className="c5"
              >
                <span>
                  INTERN
                </span>
                <p>
                  eric has bachelor of arts in communication science so we let him talk to our clients. other than that, we only really see him at after-work drinks. Main responsibilities:
                </p>
                <ul>
                  <li>
                    none
                  </li>
                  <br />
                </ul>
                <SocialBar>
                  <social-barstyles__Container>
                    <div
                      className="c6"
                    >
                      <social-barstyles__IconContainer
                        aria-label="Link to Github"
                        href="https://github.com/eegli"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Github"
                          className="c7"
                          href="https://github.com/eegli"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaGithub>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 496 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 496 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaGithub>
                        </a>
                      </social-barstyles__IconContainer>
                      <social-barstyles__IconContainer
                        aria-label="Link to Linkedin"
                        href="https://www.linkedin.com/in/ericegli/"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Linkedin"
                          className="c7"
                          href="https://www.linkedin.com/in/ericegli/"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaLinkedinIn>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 448 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 448 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaLinkedinIn>
                        </a>
                      </social-barstyles__IconContainer>
                      <social-barstyles__IconContainer
                        aria-label="Link to Instagram"
                        href="http://instagram.com/eric.film"
                        rel="noopener"
                        target="_blank"
                      >
                        <a
                          aria-label="Link to Instagram"
                          className="c7"
                          href="http://instagram.com/eric.film"
                          rel="noopener"
                          target="_blank"
                        >
                          <FaInstagram>
                            <IconBase
                              attr={
                                Object {
                                  "viewBox": "0 0 448 512",
                                }
                              }
                            >
                              <svg
                                fill="currentColor"
                                height="1em"
                                stroke="currentColor"
                                strokeWidth="0"
                                style={
                                  Object {
                                    "color": undefined,
                                  }
                                }
                                viewBox="0 0 448 512"
                                width="1em"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                                  key="0"
                                />
                              </svg>
                            </IconBase>
                          </FaInstagram>
                        </a>
                      </social-barstyles__IconContainer>
                    </div>
                  </social-barstyles__Container>
                </SocialBar>
              </div>
            </cardstyles__CardDescription>
          </div>
        </cardstyles__Card>
      </div>
    </cardstyles__CardWrapper>
  </CardComponent>
   
</ThemeProvider>
`;
